monitor/event.go

```diff
@@ -90,6 +93,16 @@ func (e EventWrapper) ParseEvent(ethClient *mainchain.EthClient) interface{} {
 	return res
 }
 
+func eventCheckInterval(name EventName) uint64 {
+	m := viper.GetStringMap(common.FlagEthCheckInterval)
+	if m[string(name)] != nil {
+		return uint64(m[string(name)].(float64))
+	} else if m[strings.ToLower(string(name))] != nil {
```

Comment:
why this is necessary?

Reply:
If you take a look at the copied config.json in the sgn root folder for testing, you can see that all names are converted to lowercases, I guess it's changed by the viper package?